Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -3764,6 +3783,27 @@ void drawMaxAccelMenu() {
 
 #endif
 
+#if ENABLED(EDITABLE_HOMING_FEEDRATE)
+
+  void drawHomingFRMenu() {
+    checkkey = ID_Menu;
+    if (SET_MENU(homingFRMenu, MSG_HOMING_FEEDRATE, 4)) {
+      BACK_ITEM(drawMotionMenu);
+      #if HAS_X_AXIS
+        EDIT_ITEM(ICON_HomeX, MSG_HOMING_FEEDRATE_X, onDrawPFloatMenu, setHomingX, &homing_feedrate_mm_m.x);
```

Comment:
I've done something similar for mine, in the end i used 
`uint32_t xhome = MMM_TO_MMS(homing_feedrate_mm_m.x);
EDIT_ITEM(ICON_MaxAccX, MSG_VMAX_A, onDrawPInt32Menu, setHomeSpeedX, &xhome);`
to display it and
`void applyHomeSpeed() { updateHomingSpeed((AxisEnum)hmiValue.select, MMS_TO_MMM(menuData.value)); }`

`void setHomeSpeedX() { hmiValue.select = X_AXIS; setIntOnClick(min_feedrate_edit_values.x, max_feedrate_edit_values.x, MMM_TO_MMS(homing_feedrate_mm_m.x), applyHomeSpeed); }`
to apply it.

Works for me even if its not the tidiest solution.

Reply:
that was my thought as well, or sort of workaround/middle-man code between values. i suppose something like this would have to do, unless theres a way to explicitly convert.

Reply:
intstead of `hmiValue.select = X_AXIS` you can use `hmiValue.axis = X_AXIS`

Reply:
@Crazy-Charles I was able to simplify the code to just this:

```y
#if HAS_X_AXIS
  void setHomingX() { setPIntOnClick(min_homing_edit_values.x, max_homing_edit_values.x); }
#endif

...

#if HAS_X_AXIS
  static uint16_t xhome = static_cast<uint16_t>(homing_feedrate_mm_m.x);
  EDIT_ITEM(ICON_MaxSpeedJerkX, MSG_HOMING_FEEDRATE_X, onDrawPIntMenu, SetHomingX, &xhome);
#endif
```

Reply:
Sorry but I actually have no idea what i'm doing when it comes to writing code or the best way to do things, generally if it does what i want it to do then that is good enough for me.

Reply:
actually I forgot about this part:
```y
void applyHomingFR() { updateHomingFR(HMI_value.axis, MenuData.Value); }

void setHomingX() { hmiValue.axis = X_AXIS; setIntOnClick(min_homing_edit_values.x, max_homing_edit_values.x, homing_feedrate_mm_m.x, applyHomingFR); }

```

this *should* work. I at least got the display part correct, just need to actually test. in this I didnt bother converting `MMS_MMM` or vice-versa.